src/js/components/__tests__/__snapshots__/components-test.js.snap

```diff
@@ -1247,6 +1247,15 @@ exports[`Components loads 1`] = `
           "margin": {
             "bottom": "small",
           },
+          "survey": {
+            "label": {
+              "margin": {
+                "vertical": "xsmall",
+              },
+              "size": "large",
+              "weight": 500,
```

Comment:
Figma seems to be using default weight (400) -- would be good to double check the desired value.

Reply:
yeah you are correct and `medium` these values were more just to show it works since I would take them out anyways here and put in hpe theme